rest-service/manager_rest/test/endpoints/test_labels.py

```diff
@@ -86,7 +86,7 @@ def test_remove_resource_labels(self):
         resource = self.put_resource_with_labels(self.LABELS)
         self.assert_resource_labels(resource.labels, self.LABELS)
         updated_res = self.update_resource_labels(resource.id, [])
-        assert not updated_res.labels
+        assert updated_res.labels == []
```

Comment:
Do you want to do this on the other ones from the first commit too?

Reply:
OK